File: coresuite-dotnet-framework-console-cs/generator/DigitalSignatures.cs
using ceTe.DynamicPDF;
using ceTe.DynamicPDF.PageElements.Forms;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace coresuite_dotnet_framework_console_cs.Generator
{
    class DigitalSignatures
    {
        public static void Run(string outputPdfPath)
        {
            Document document = new Document();
            Page page = new Page();

            Signature signature = new Signature("SigField", 10, 10, 250, 100);
            page.Elements.Add(signature);
            document.Pages.Add(page);

            Certificate certificate = new Certificate(Util.GetResourcePath("Data/JohnDoe.pfx"), "dynamicpdf");

            document.Sign("SigField", certificate);

            document.Draw(outputPdfPath);
        }
    }
}

File: coresuite-dotnet-framework-console-vb/generator/DigitalSignatures.vb
Imports ceTe.DynamicPDF
Imports ceTe.DynamicPDF.PageElements.Forms

Public Class DigitalSignatures
    Public Shared Sub Run(outputPdfPath As String)
        Dim MyDocument As ceTe.DynamicPDF.Document = New ceTe.DynamicPDF.Document
        Dim MyPage As ceTe.DynamicPDF.Page = New ceTe.DynamicPDF.Page
        'Create & add Signature Form Field
        Dim MySignature As New Signature("SigField", 10, 10, 250, 100)
        MyPage.Elements.Add(MySignature)
        MyDocument.Pages.Add(MyPage)

        Dim MyCertificate As New Certificate(Util.GetResourcePath("Data/JohnDoe.pfx"), "dynamicpdf")
        MyDocument.Sign("SigField", MyCertificate)

        MyDocument.Draw(outputPdfPath)
    End Sub
End Class
